src/utils/throwable.js

```diff
@@ -0,0 +1,9 @@
+// some popup with error message
+
+export async function throwable(callback) {
+  try {
+    return await callback();
+  } catch (error) {
+    alert(error.message);
```

Comment:
add a custom pop-up window.
let `throwable` receive a second argument with settings for that pop-up such as:
- position - top left, bottom right, etc
- type - warning, error, note - it should effect the color of the pop-up.
if you come up with more options that's okay too